<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Importar listas</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="0" column="2">
              <widget class="QToolButton" name="toolButton_3">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Arquivo</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QToolButton" name="toolButton_2">
               <property name="text">
                <string>x</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>200</horstretch>
           <verstretch>255</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>350</height>
          </size>
         </property>
         <property name="title">
          <string>Midias</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QListWidget" name="listWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="lineWidth">
       <number>10</number>
      </property>
      <property name="text">
       <string>m3uPlayer ............</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="2" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="toolButton_4">
          <property name="text">
           <string>P</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Pesquisar: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Reprodução:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QToolButton" name="toolButton">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="pageStep">
             <number>8</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0" rowspan="2" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Iniciar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Parar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string>Anterior</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Próximo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Volume </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>70</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>% </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menum3uPlayer">
    <property name="title">
     <string>m3uPlayer</string>
    </property>
    <addaction name="actionAbrir_arquivo_m3u"/>
    <addaction name="actionFechar_arquivo_m3u"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuOp_es">
    <property name="title">
     <string>Opções</string>
    </property>
    <addaction name="actionPesquisar_na_lista"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Sobre</string>
    </property>
    <addaction name="actionDesenvolvimento"/>
   </widget>
   <addaction name="menum3uPlayer"/>
   <addaction name="menuOp_es"/>
   <addaction name="menuSobre"/>
  </widget>
  <action name="actionAbrir_arquivo_m3u">
   <property name="text">
    <string>Abrir arquivo (.m3u)</string>
   </property>
  </action>
  <action name="actionFechar_arquivo_m3u">
   <property name="text">
    <string>Fechar arquivo (.m3u)</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionPesquisar_na_lista">
   <property name="text">
    <string>Pesquisar na lista</string>
   </property>
  </action>
  <action name="actionDesenvolvimento">
   <property name="text">
    <string>Desenvolvimento</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
